bottlecap/src/appsec/processor/context.rs

```diff
@@ -358,14 +507,53 @@ impl Context {
         }
     }
 }
+impl Drop for Context {
+    fn drop(&mut self) {
+        // First off, gentle log nudge -- we should have marked the context's response as seen before this can drop...
+        if !self.response_seen {
+            debug!(
+                "aap: Context being dropped without the response being marked as seen, this may cause traces to be dropped"
+            );
+        }
+        // In debug assertions mode, hard-crash if it means we're effectively dropping a trace.
+        debug_assert!(
```

Comment:
Can we ensure a panic won't crash the Extension, or make it idle for the next invocations?